libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -23,6 +23,17 @@ export interface BucketProps {
   readonly public?: boolean;
 }
 
+/**
+ * Options for `Bucket.put`.
+ */
+export interface BucketPutProps {
```

Comment:
We need to decide on a naming convention for these option structs. I think we normally call them `XxxOptions` if they are not initialization props.

@Chriscbr what do you think?

Reply:
I was also going to bring this up. Actually, @Chriscbr proposed to call the struct `BucketPutOptions`, but then I saw `SetConsumerProps` used for `Queue`, `ServiceOnStartProps` used for `Service`, etc. so I wanted to follow the convention. The only place where I introduced the word `Option` is inside `Json`.